project.clj

```diff
@@ -64,6 +66,9 @@
                  [hiccup "1.0.5"]                                     ; HTML templating
                  [honeysql "0.8.2"]                                   ; Transform Clojure data structures to SQL
                  [io.crate/crate-jdbc "2.1.6"]                        ; Crate JDBC driver
+                 [io.forward/yaml "1.0.6"                             ; Clojure wrapper for SnakeYAML
```

Comment:
what's a SnakeYAML?

Reply:
The YAML parser we're already using (see line 98). This is just a super lightweight Clojure wrapper.

Reply:
Got it. It would be nice to add that to the comment next to the dependency so no one else has to wonder the same thing :wink:

Reply:
Fixed.